wg-reliability/sippy/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
```

Comment:
We also want Ingress and Certificate too, as e.g. here:
https://github.com/kubernetes/k8s.io/tree/master/perf-dash.k8s.io